build.sh

```diff
@@ -17,39 +17,39 @@ fi
 
 # code will be compiled in this container
 BUILDER_IMAGE='quay.io/geonet/golang:1.13.1-alpine'
-RUNNER_IMAGE='quay.io/geonet/alpine:3.10'
-
+RUNNER_IMAGE='quay.io/geonet/go-scratch:latest'
+DOCKERFILE=${DOCKERFILE:-"Dockerfile"}
```

Comment:
This makes local build/test somehow inconvenient (we can simply type `./build.sh` to build for local test) .

The existing way is good enough.

Reply:
@ardrigh how do you think?